src/test/groovy/org/prebid/server/functional/tests/privacy/GdprAuctionSpec.groovy

```diff
@@ -314,7 +315,7 @@ class GdprAuctionSpec extends PrivacyBaseSpec {
         tcfPolicyVersion << [TCF_POLICY_V2, TCF_POLICY_V4, TCF_POLICY_V5]
     }
 
-    def "PBS auction should reject request with proper warning when incoming consent.tcfPolicyVersion have invalid parameter"() {
+    def "PBS auction should reject request with proper warning and metrics when incoming consent.tcfPolicyVersion have invalid parameter"() {
         given: "Tcf consent string"
         def invalidTcfPolicyVersion = PBSUtils.getRandomNumber(5, 63)
         def tcfConsent = new TcfConsent.Builder()
```

Comment:
The current change should allow us to accept any value for `tcfPolicyVersion`, and we should receive `response.seatbid`. Please change `tcfConsent` to:

```
def tcfConsent = new TcfConsent.Builder()
        .setPurposesLITransparency(BASIC_ADS)
        .setTcfPolicyVersion(invalidTcfPolicyVersion)
        .setVendorLegitimateInterest([GENERIC_VENDOR_ID])
        .build()
```
Additionally, add test variants where `invalidTcfPolicyVersion` is set to `5`, random value, and max value - 63. Ensure that we include responses with request, as we no longer reject them

Reply:
Good catch